src/components/Instance/Definitions/stories.js

```diff
@@ -0,0 +1,18 @@
+import { withKnobs, object } from "@storybook/addon-knobs";
+
+import Definitions from '.';
+
+export default { title: "Definitions", decorators: [withKnobs]};
+
+export const basic = () => ({
+  Component: Definitions,
+  props: {
+    definition: object('definitions', {
+      definitions: { 
+        "A": {id:'A'},
+        "B": {id:'B'}
+      }
+    })
```

Comment:
You had one level of definitions too many.

```suggestion
    definition: object('definitions', {
        "A": {id:'A'},
        "B": {id:'B'}
    })
```